dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/jobService/data/CancelableBlobStrategy.kt

```diff
@@ -0,0 +1,30 @@
+package com.spectralogic.dsbrowser.gui.services.jobService.data
+
+import com.spectralogic.ds3client.helpers.JobPart
+import com.spectralogic.ds3client.helpers.strategy.blobstrategy.BlobStrategy
+import com.spectralogic.ds3client.helpers.strategy.blobstrategy.PutSequentialBlobStrategy
+import javafx.beans.property.BooleanProperty
+import javafx.beans.property.SimpleBooleanProperty
+import javafx.beans.value.ObservableBooleanValue
+import java.security.Provider
+import java.util.concurrent.atomic.AtomicBoolean
+import java.util.function.Supplier
+
+class CancelableBlobStrategy(val strategy: BlobStrategy, val cancelled: Supplier<Boolean>) : BlobStrategy by strategy {
```

Comment:
Is this class needed anymore with the updates to the sdk?

Reply:
I'll delete it